src/main/kotlin/org/jetbrains/kotlinx/jupyter/util.kt

```diff
@@ -68,3 +68,43 @@ fun Int.toSourceCodePositionWithNewAbsolute(code: SourceCode, newCode: SourceCod
 fun ResultsRenderersProcessor.registerDefaultRenderers() {
     register(bufferedImageRenderer)
 }
+
+/**
+ * Stores info about where a variable Y was declared and info about what are they at the address X.
+ * T: key, stands for a way of addressing variables, e.g. address.
+ * V: value, from Variable, choose any suitable type for your variable reference.
+ * Default: Int, String
```

Comment:
T = Int, V = String

Reply:
BTW, key type is usually named `K`, not `T`

Reply:
T and V is usually combo from Java, but ok